docs/access-member-clusters-through-karmada.md

```diff
@@ -0,0 +1,258 @@
+```
+
+> Note: For the member cluster that joins karmada in pull mode,  we can deploy apiserver-network-proxy(ANP) to access it.
+
+## Unified authentication
+
+For one or a group of user subjects(users, groups, or service accounts ) in a member cluster, we can import it into karmada control plane and grant it the `clusters/proxy` permission, so that we can access the member cluster with permission of the user subject through karmada.
+
+In this section, we use a serviceaccount named `zhangsan` for the test.
+
+### Step1: Create ServiceAccount in member1 cluster
+
+Create a serviceaccount that does not have any permission:
+
+```
+kubectl --kubeconfig /root/.kube/members.config --context member1 create serviceaccount zhangsan
+```
+
+### Step2: Create ServoceAccpimt in karmada control plane
+
+```
+kubectl --kubeconfig /root/.kube/karmada.config --context karmada-apiserver create serviceaccount zhangsan
+```
+
+In order to grant serviceaccount the `clusters/proxy` permission, apply the following rbac yaml file:
+
+cluster-proxy-rbac.yaml:
+
+<details>
+
+<summary>unfold me to see the yaml</summary>
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-proxy-clusterrole
+rules:
+- apiGroups:
+  - 'cluster.karmada.io'
+  resources:
+  - clusters/proxy
+  resourceNames:
+  - member1
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-proxy-clusterrolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-proxy-clusterrole
+subjects:
+  - kind: ServiceAccount
+    name: zhangsan
+    namespace: default
+  # The token generated by the serviceaccount can parse the gourp information. Therefore, here need to specify below group information.
+  - kind: Group
+    name: "system:serviceaccounts"
+  - kind: Group
+    name: "system:serviceaccounts:default"
+  - kind: Group
+    name: "system:authenticated"   
+```
+
+</details>
+
+```
+kubectl --kubeconfig /root/.kube/karmada.config --context karmada-apiserver apply -f cluster-proxy-rbac.yaml
+```
+
+### Step3: Access member1 cluster
+
+Obtain token of serviceaccount `zhangsan`:
+
+```
+kubectl get secret `kubectl get sa zhangsan -oyaml | grep token | awk '{print $3}'` -oyaml | grep token: | awk '{print $2}' | base64 -d
+```
+
+Then construct a kubeconfig file `zhangsan.config` for `zhangsan` serviceaccount:
+
+```yaml
+apiVersion: v1
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
+    server: {karmada-apiserver-address} # replace {karmada-apiserver-address} with karmada-apiserver-address, you can find it in /root/.kube/karmada.config file.
+  name: zhangsan
+contexts:
+- context:
+    cluster: zhangsan
+    user: zhangsan
+  name: zhangsan
+current-context: zhangsan
+kind: Config
+users:
+- name: zhangsan
+  user:
+    token: {token} # replace {token} with the token obtain above.
+```
+
+Run as below to access member1 cluster:
```

Comment:
```suggestion
Run the command below to access member1 cluster:
```

Reply:
We can say "use the command that runs as below", but when the subject is you/user, the sentence becomes "you run as below". Logically doesnt make sense.